stream/src/placeholder/instance.rs

```diff
@@ -2,14 +2,14 @@ use super::{
     handshake::{create_handshake, Handshake, IncomingHandshake},
     utils::{
         codec::{recv_frame, send_frame},
-        nonce::encode,
+        nonce::NonceInfo,
```

Comment:
nit: Since there is only one type of `Info` in nonce, I recommend calling this just `Info`/`Instance`/`Nonce` rather than `NonceInfo`.

(This is probably more of a reflection of my preferred style but I tend to use the "import path" to name things/rename on import rather than encoding uniqueness into the struct/function name itself to keep the code less verbose):
`use nonce::Instance as NonceInfo` OR `use nonce` -> `nonce::Info{}` rather than `use nonce::NonceInfo`